### PR TITLE
Feature/documentation

### LINKED ISSUE #450: document contract deployment and upgrade more detailed
tbd

relates to: https://github.com/oceanprotocol/ocean/issues/289

### PR DESCRIPTION
**Pull request by H34D**
## Description

Updated RELEASE_PROCESS.md
Updated README.MD
Added description on how to use the `npm` package of `keeper-contracts`.

## Is this PR related with an open issue?

Closes #450 #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # keeper-contracts
 
-> 💧 Integration of TCRs, CPM and Ocean Tokens in Solidity
+> 💧 Integration of SEAs, DID and OceanToken in Solidity
```
Comment (ttmc) [inline README.md]: I updated the corresponding line near the top of the repository home page.

Comment (H34D) [reply README.md]: thanks troy

Comment (aabdulwahed) [reply README.md]: @H34D and @ttmc it would be great if you can update the repo description as well 
image

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -1,46 +1,133 @@
...
+##### Persist artifacts
+
+- Commit all changes in `artifacts/*.nile.json`
+
+#### Kovan
+
+- Copy the wallet file for `kovan` > `cp wallets_kovan.json wallets.json`
+- run `export MNEMONIC=<your kovan mnemonic>`. You will find them in the password manager.
+- run `export INFURA_TOKEN=<your infura token>`. You will get it from `infura`.
+
```
Comment (aabdulwahed) [inline RELEASE_PROCESS.md]: The same here for `Gnonsis` wallet, you will need to import the same MNEMONIC or the seed phrase to `metamask`.

File: RELEASE_PROCESS.md [hunk 3]
```diff
@@ -1,46 +1,133 @@
...
+- run `npm run clean` to clean the work dir.
+- run `npm run compile` to compile the contracts.
+
+#### Nile
+
+- Copy the wallet file for `nile` 
+  - `cp wallets_nile.json wallets.json`
+- run `export MNEMONIC=<your nile mnemonic>`. You will find them in the password manager.
+
```
Comment (aabdulwahed) [inline RELEASE_PROCESS.md]: Also, you will need to import seed phrase to your `metamask`. As you will need to confirm transactions using `Gnosis` wallet.

File: RELEASE_PROCESS.md [hunk 4]
```diff
@@ -1,46 +1,133 @@
...
+- Commit all changes in `artifacts/*.kovan.json`
+
+#### Approve upgrades
+
+All upgrades of the contracts have to be approved by a the `upgrader` wallet configured in the `wallets.json` file.
+
+- go to https://wallet.gnosis.pm
+- Load `upgrqder` wallet
+- Confirm
```
Comment (aabdulwahed) [inline RELEASE_PROCESS.md]: switch your metamask to the wallet accounts and confirm the transaction. Finally Execute transaction.

**Comment by aabdulwahed** [comment]
@H34D this also need to be updated: https://github.com/oceanprotocol/keeper-contracts/blob/release/v0.9/doc/files/Keeper-Contracts.png

File: RELEASE_PROCESS.md (hunk 2 above)
Comment (H34D) [reply RELEASE_PROCESS.md]: well, no. The deployer has not to be a member of the multi sig wallet. Only the upgrader has to.

File: RELEASE_PROCESS.md (hunk 3 above)
Comment (H34D) [reply RELEASE_PROCESS.md]: For upgrade yes, for deploy no. I will add additional informations about the roles.

File: RELEASE_PROCESS.md (hunk 4 above)
Comment (H34D) [reply RELEASE_PROCESS.md]: will add that, and fix the typo.

**Comment by H34D** [comment]
> contracts/blob/release/v0.9/doc/files/Keeper-Contracts.png

please open an issue, i did this for the other documents.

File: RELEASE_PROCESS.md (hunk 2 above)
Comment (aabdulwahed) [reply RELEASE_PROCESS.md]: okay thanks for clarification.

**Comment by aabdulwahed** [comment]
>> please open an issue, i did this for the other documents as well.

Reported here [#470](https://github.com/oceanprotocol/keeper-contracts/issues/470)

**Pull request merged**